classes/01class/exercises/c01-git03/mendoncaj/README.md

```diff
@@ -0,0 +1,39 @@
+# c01-git03
+
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+
+> Yes, before adding, the big file the size was 100K however after commit the size was increased to 20M and then after `git revert HEAD` the size remains within 10M. 
```

Comment:
Creating a bigger file like 100M would demonstrate it better, but the this already demonstrated the problem.